eng/pipelines/templates/steps/typespec-ci.yml

```diff
@@ -5,6 +5,18 @@ parameters:
   type: string
 
 steps:
-- script: npx --no tsv ${{parameters.Folder}}
+- bash: |
```

Comment:
bash is ok for now, but eventually I'll incorporate logic this into the auto-detection powershell script

Reply:
I haven't looked at the auto-detect pwsh script yet, but I think we should try to keep that tool as generic as possible and not add features related to git restore/clean.